internal/cli/tiltfile_result.go

```diff
@@ -42,17 +50,19 @@ func newTiltfileResultCmd() *tiltfileResultCmd {
 func (c *tiltfileResultCmd) register() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "tiltfile-result",
-		Short: "Exec the Tiltfile and print results as JSON (note: the API is unstable and may change)",
-		Long: `Exec the Tiltfile and print results as JSON (note: the API is unstable and may change).
+		Short: "Exec the Tiltfile and print data about execution. By default, prints Tiltfile execution results as JSON (note: the API is unstable and may change); can also print timings of Tiltfile Builtin calls.",
+		Long: `Exec the Tiltfile and print data about execution. By default, prints Tiltfile execution results as JSON (note: the API is unstable and may change); can also print timings of Tiltfile Builtin calls.
 
-Exit code 0: successful Tiltfile evaluation (JSON printed to stdout)
+Exit code 0: successful Tiltfile evaluation (data printed to stdout)
 Exit code 1: some failure in setup, printing results, etc. (any logs printed to stderr)
 Exit code 5: error when evaluating the Tiltfile, such as syntax error, illegal Tiltfile operation, etc. (any logs printed to stderr)
 
 Run with -v | --verbose to print Tiltfile execution logs on stderr, regardless of whether there was an error.`,
 	}
 
 	addTiltfileFlag(cmd, &c.fileName)
+	cmd.Flags().BoolVarP(&c.builtinTimings, "builtin-timings", "b", false, "If true, print timing data for Tiltfile builtin calls instead of Tiltfile result JSON")
+	cmd.Flags().StringVar(&c.durThreshold, "dur-threshold", "", "Only compatible with Builtin Timings mode. Should be a Go duration string. If passed, only print information about builtin calls lasting this duration and longer.")
```

Comment:
there's actually a builtin duration flag that will do this for you!
https://golang.org/pkg/flag/#DurationVar
https://pkg.go.dev/github.com/spf13/pflag?tab=doc#DurationVar

Reply:
🙌 🎉